kythe/cxx/common/kzip_writer.cc

```diff
@@ -84,6 +97,14 @@ KzipWriter::~KzipWriter() {
 
 StatusOr<std::string> KzipWriter::WriteUnit(
     const kythe::proto::IndexedCompilation& unit) {
+  if (!initialized_) {
+    auto status = InitializeArchive(archive_);
+    if (!status.ok()) {
+      return status;
+    } else {
```

Comment:
Remove `else` since the first branch escapes the method?  That's go style.  I'm not sure if it is C++.